Cargo.toml

```diff
@@ -16,11 +16,11 @@ path = "src/prometheus.rs"
 [dependencies]
 clap = { version = "3.2" }
 colored = "2"
-prettytable-rs = "0.8.0"
-nix = "0.22.0"
-tokio = { version = "0.2", features = ["macros", "rt-threaded", "time"] }
+prettytable-rs = "0.10.0"
+nix = "0.26.0"
+tokio = { version = "1", features = ["full"] }
```

Comment:
Don't see any issue. Bear in kind that transitioning tokio to `full` changes way it manage thread and ship by default multi-threaded runtime with thread pool.

Tokio features for ref: <https://docs.rs/tokio/latest/tokio/?search=runtime#feature-flags>

If you want the same as `["macros", "rt-threaded", "time"]` for 1.0.0 tokio it will be `["macros", "rt-multi-thread", "time]`.

May to change main tokio proc macro use current thread